organize/views.py

```diff
@@ -1,3 +1,57 @@
-from django.shortcuts import render
+from formtools.wizard.views import NamedUrlSessionWizardView
+from django.shortcuts import redirect, render
 
-# Create your views here.
+from .forms import (
+    PreviousEventForm, ApplicationForm, WorkshopForm, OrganizersForm)
+
+# ORGANIZE FORM #
+
+FORMS = (("previous_event", PreviousEventForm),
+         ("application", ApplicationForm),
+         ("organizers", OrganizersForm),
+         ("workshop", WorkshopForm))
+
+TEMPLATES = {"previous_event": "organize/form/step1_previous_event.html",
+             "application": "organize/form/step2_application.html",
+             "organizers": "organize/form/step3_organizers.html",
+             "workshop": "organize/form/step4_workshop.html"}
+
+
+class OrganizeFormWizard(NamedUrlSessionWizardView):
+
+    def get_template_names(self):
+        return [TEMPLATES[self.steps.current]]
+
+    def done(self, form_list, **kwargs):
+        print("FINISHED")
```

Comment:
This is a debug print, right? If we want to keep it in production, using logging would be better.